app/views/arts/build/upload_photos.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="wizard-form">
+  <%= form_for @art, method: :put, url: wizard_path do |f| %>
+    <% if f.object.errors.any? %>
+      <div class="error_messages">
+        <% f.object.errors.full_messages.each do |error| %>
+          <p><%= error %></p>
+        <% end %>
+      </div>
+    <% end %>
+    <h1>Click the button to upload pictures</h1>
+
+    <div data-controller="photo-uploader">
+      <div class="photos">
+        <%= f.file_field :photos, :multiple => true, :data => {:action => "photo-uploader#addFile"}, title: "This is were your photo will be" %>
```

Comment:
`multiple: true` instead of `:multiple => true`
And `data: { action: "photo-uploader#addFile" }`

How come there's no event attached to your action ? 🤔